Adafruit_Si7021.h

```diff
@@ -62,13 +62,20 @@ class Adafruit_Si7021 {
   void reset(void);
   void readSerialNumber(void);
   float readHumidity(void);
-	const char *getModel(void) { return _model; };
+  const char *getModel(void);
```

Comment:
I would prefer:
```c++
sensorType getModel(void);
```

And for print/debug a function with string as return, maybe named:
```c++
const char *getModelDebug(void); 
const char *getModelString(void); 
const char *getModelPrintable(void);